libpromises/loading.c

```diff
@@ -111,6 +111,14 @@ Policy *Cf3ParseFile(const GenericAgentConfig *config, const char *input_path)
         }
 
         policy = PolicyFromJson(json_policy);
+        if (policy == NULL)
+        {
+            Log(LOG_LEVEL_ERR, "Failed to deserialize a policy from the JSON "
+                "input file '%s'", input_path);
```

Comment:
Try to wrap but not break string literals, because it reduces "grepability" of the codebase. I think there are some relevant guidelines in HACKING.md.

Reply:
Good point, done.